documentation/modules/upgrading/con-upgrade-custom-resources.adoc

```diff
@@ -0,0 +1,107 @@
+// This module is included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='con-upgrade-custom-resources-{context}']
+
+= API versioning
+
+[role="_abstract"]
+Custom resources are edited and controlled using APIs added to Kubernetes by CRDs.
+Put another way, CRDs extend the Kubernetes API to allow creation of custom resources.
+CRDs are themselves resources within Kubernetes.
+They are installed in a Kubernetes cluster to define the versions of API for the custom resource.
+Each version of the custom resource API can define its own schema for that version.
+
+For release {ProductVersion}, Strimzi custom resources must specify `{ApiVersion}` as the API version.
+Strimzi Operators access the custom resources served by the Kubernetes API server using a URL path (_API path_) which includes the API version.
```

Comment:
I would remove this and replace it with just something like _You should use v1beta2 from now on._

Reply:
This is outdated. See screen capture on https://github.com/strimzi/strimzi-kafka-operator/pull/4492#discussion_r588419934 to see if we need further chnage